.travis.yml

```diff
@@ -1,11 +1,20 @@
+dist: bionic
+
 language: go
 
 go:
 - 1.13.4
 
+before_script:
+  - echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_18.04 /" | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+  - curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_18.04/Release.key | sudo apt-key add -
+  - sudo apt-get update -qq
+  - sudo apt-get -qq -y install podman slirp4netns
```

Comment:
Why do we need to install podman, does `make release` need it?

Reply:
@anjannath yes make release need the podman